go.mod

```diff
@@ -74,3 +76,5 @@ require (
 )
 
 replace github.com/onsi/gomega => github.com/onsi/gomega v1.16.0
+
+replace github.com/containers/common => github.com/n1hility/common v0.46.1-0.20211209203754-7a821838ac56
```

Comment:
@n1hility are there any specific changes in `c/common` which are not present in upstream. 
Could we sync it with upstream.

Reply:
Yes sorry I must have accidentally reverted the switch to upstream in rebasing. This is fixed now.